### PR TITLE
FIX: better handling of invite links after they are redeemed

### PR DESCRIPTION
**Pull request by techAPJ**
FIX: deprecate invite_passthrough_hours setting

### CODE REVIEW

File: config/site_settings.yml [hunk 1]
```diff
@@ -408,7 +408,6 @@ users:
     default: true
   invite_expiry_days:
     default: 30
-  invite_passthrough_hours: 0
```
Comment (tgxworld) [inline config/site_settings.yml]: We need to write a migration to drop site settings in the db here

**Comment by tgxworld** [comment]
@techAPJ I think we should update the commit message to state that we're removing that site setting.

**Comment by techAPJ** [comment]
@tgxworld I updated the commit message already... see https://github.com/discourse/discourse/pull/5809/commits/807200dc054d7a46edc6fe9395f0ff14497fceb1

Just the PR description was not updated, will do so now.

File: config/site_settings.yml (hunk 1 above)
Comment (techAPJ) [reply config/site_settings.yml]: Done.

File: db/migrate/20180508142711_remove_invite_passthrough_hours.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveInvitePassthroughHours < ActiveRecord::Migration[5.1]
+  def change
```
Comment (tgxworld) [inline db/migrate/20180508142711_remove_invite_passthrough_hours.rb]: @techAPJ This should be `up` only as you can't rollback the SQL statement.

Comment (techAPJ) [reply db/migrate/20180508142711_remove_invite_passthrough_hours.rb]: Good catch. Fixed in: https://github.com/discourse/discourse/commit/27efa4580038617c1a043aca17df8d2ca09ba746.

**Pull request merged**